charts/kong-collectorapi/values.yaml

```diff
@@ -8,23 +8,6 @@ waitImage:
   tag: latest
   pullPolicy: IfNotPresent
 
-# These parameters enable collector to send requests to Kong Admin API
-kongAdmin:
-  # Set this to the Kong Admin API hostname
```

Comment:
I do think this might still be necessary.  Immunity is hitting kong_admin whenever it needs workspace names,

Reply:
But I can be convinced otherwise.